### PR TITLE
Feature/extensions

### LINKED ISSUE #12: Messenger Extensions support
This is long overdue but also with the release of Chat Extensions there needs to be support for messenger extensions in menu items and URL buttons.

There might be a bit more to do: https://developers.facebook.com/docs/messenger-platform/guides/chat-extensions

### PR DESCRIPTION
**Pull request by kirunchik**
Fixes #12

### CODE REVIEW

**Comment by kirunchik** [comment]
Davids tests upped the coverage, but that was done before the branch was created, so the coverage  up/down report is broken :(

File: src/message/buttons/urlButton.js [hunk 1]
```diff
@@ -35,7 +35,13 @@ export default class UrlButton extends Button {
         return this;
     }
 
+    setMessengerExtensions(value = false) {
```
Comment (radTuti) [inline src/message/buttons/urlButton.js]: can we switch this to `useMessengerExtensions` with use `true` as the default value

Comment (kirunchik) [inline src/message/buttons/urlButton.js]: We should probably then change it to useMessengerExtensions, if thats true, and disable if we do not need it

Comment (radTuti) [inline src/message/buttons/urlButton.js]: since it's not a required field, just `enableMessengerExtensions` should be fine

**Pull request merged**